.github/workflows/contract-size.yml

```diff
@@ -0,0 +1,27 @@
+name: Contract sizes
+
+on: [push]
+
+jobs:
+  size:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        package:
+          [core-contracts, core-modules, synthetix-main]
+        include:
+          - package: core-contracts
+          - package: core-modules
+          - package: synthetix-main
```

Comment:
This is not necessary on this case because it's not adding any custom variables per matrix item. (On [the other PR](https://github.com/Synthetixio/synthetix-v3/pull/553/files) we were adding  the var `coverage-file` pero item.)
```suggestion
```